src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Overview/ThorchainSaversOverview.tsx

```diff
@@ -157,11 +141,55 @@ export const ThorchainSaversOverview: React.FC<OverviewProps> = ({
     selectUnderlyingStakingAssetsWithBalancesAndIcons(state, opportunityDataFilter),
   )
 
+  const txsFilter = useMemo(
+    () => ({
+      accountId,
+      assetId,
+    }),
+    [accountId, assetId],
+  )
+  const pendingTxsLength = useAppSelector(state => selectTxsByFilter(state, txsFilter)).filter(
+    tx => tx.status === TxStatus.Pending,
```

Comment:
wondering, in the rare case a user has another pending/stuck tx (e.g. low gas on evm), are we able to filter this tighter, say pending tx's that contain a memo that includes something unique like `::ss`?

Reply:
as discussed nvm this is already discriminated enough